File: 100-Days-Series/Day 96/SumOfNegativeNumbers.py

# Function to find the sum of negative numbers in the array
def get_sum_of_negative_numeber(arr):
    sum=0
    for i in range(0,len(arr)):
        if arr[i]<0:
            sum+=arr[i]

    return sum

# Let's Start Here
arr=[5,4,-6,-7,-9,1,]
result=get_sum_of_negative_numeber(arr)
print('Sum of Negative Numbers : ',result)

File: 100-Days-Series/Day 96/SumOfNegativeNumbers.js

// Function to find the sum of negative numbers in the array
const getSumOfNeagtiveNumber=(arr)=>{
    let sum=0;
    for(let i=0;i<arr.length;i++){
        if(arr[i]<0){
            sum+=arr[i];
        }
    }
    return sum;
}
// Let's Start Here
const arr=[5,-4,3,-2,1,-6];
let result=getSumOfNeagtiveNumber(arr);
console.log('Sum of Negative Numbers : '+result);